util-taglib/src/com/liferay/taglib/portletext/RuntimeTag.java

```diff
@@ -267,6 +267,10 @@ public static void doTag(
 
 				PortalIncludeUtil.include(pageContext, _ERROR_PAGE);
 
+				if (_log.isWarnEnabled()) {
```

Comment:
@yuhai Please make it an error; also, what do you mean by "can't see the real error in UI" and is there any way to fix that?

Reply:
@dantewang, for example. for LPS-33201, the error can be displayed  with red as the background color in UI and web content display only display one article, this is obvious. 

For LPS-83930, because asset publish portlet can display lots of entries(web content article, blogs...), for other entries without template, they can be displayed right. It is hard to see the error message. In addition, even the error displays in UI, it lost some <div> (for example, the read background color).